sdccc/src/main/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantBicepsNormativeAnnexTest.java

```diff
@@ -46,6 +48,8 @@ void setUp() {
     @TestIdentifier(EnabledTestConfig.BICEPS_R5006)
     @TestDescription(
             "Verifies that for all incoming messages, Localized text with the @Ref attribute always has a version")
+    @RequirePrecondition(
+            manipulationPreconditions = {ManipulationPreconditions.RemoveAndReinsertDescriptorManipulation.class})
```

Comment:
I am not quite sure what exactly the purpose of this is, but if it is for getting relevant test data and possibly a violation of the requirement, I think the manipulation would have to be more specific to achieve that.